openassessment/xblock/grade_mixin.py

```diff
@@ -108,6 +108,7 @@ def render_grade_complete(self, workflow):
         has_submitted_feedback = False
 
         if "peer-assessment" in assessment_steps:
+            peer_api.get_score(submission_uuid, self.workflow_requirements()["peer"])
```

Comment:
See https://github.com/edx/edx-ora2/blob/efischer/show_peer_scores/openassessment/assessment/api/peer.py#L434 for where this filtering takes place.